src/Bowerphp/Command/Command.php

```diff
@@ -27,9 +27,13 @@
  */
 abstract class Command extends BaseCommand
 {
+    /** @var  Filesystem $filesystem */
```

Comment:
We prefer to keep full extended docblocks (just like in the rest of the code).
Also, there's no need to repeat variable name.